QA/README.md

```diff
@@ -0,0 +1,79 @@
+# CherryTwist - Quality Assurance
+
+This document provides overview of "Cherrytwist" quality assurance and testing activities. Some of them are in progress and some are to be started.
+
+# Process
+ 
+- Perform functional/non-functional testing types on reviewed "github" issues
+    - Depending on the "github" issue scope, testing is performed on different environments 
+- Log defects, communicate  and retest them
+- Perform regression testing as part of "github" issues testing
+- Automate/maintain test cases on different levels
+- Test data management
```

Comment:
Package dependency audit: regular review of reported vulnerabilities in packages dependent on and then address